include/adapter/dealii_interface.h

```diff
@@ -119,7 +116,7 @@ namespace Adapter
     interface_nodes_ids.reserve(this->mf_data->n_boundary_face_batches() * 0.5);
 
     // Set up data structures
-    FEFaceIntegrator phi(*this->mf_data, true, mf_dof_index, mf_quad_index);
+    FaceIntegrator phi(*this->mf_data, true, mf_dof_index, mf_quad_index);
```

Comment:
The cool kids write `FEFaceIntegrator` ;)

Reply:
Damn, so close. Fixed it quickly to the cool version :sunglasses: